lib/Mongo/MongoClient.php

```diff
@@ -74,6 +90,8 @@ public function __construct($server = 'default', array $options = ["connect" =>
 
         $this->server = $server;
         $this->client = new Client($server, $options, $driverOptions);
+        $info = $this->client->__debugInfo();
```

Comment:
I wouldn't rely on `__debugInfo()` providing a consistent API. I know we need the manager to provide some of the functionality but I'm sure there is a reason it is not exposed in `MongoDB\Client`. Maybe @jmikola can provide some insight here as to best proceed with this?